src/foundry/collections/worldCollection.d.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * A singleton Collection of world-level Document objects within the Foundry Virtual Tabletop.
+ * Each primary Document type has an associated subclass of WorldCollection which contains them.
+ */
+declare abstract class WorldCollection<T extends Entity = Entity> extends DocumentCollection<T> {
```

Comment:
This class has been adjusted slightly by 0.8.5, please revisit it. For example, `_source` is now `readonly`.